server/app/models/va/Ambassador.js

```diff
@@ -189,4 +189,14 @@ module.exports = {
     type: "integer",
     default: null,
   },
+  admin_bonus: {
+    //a hopefully positive number denoting how trusted the ambassador is (but it can be negative)
+    type: "integer",
```

Comment:
Instead of `type: "integer"` let's use `type: "number"`.  When you put integers in Neo4J/Neode it gives you back these awkward `{high: 123, low: 456}` objects instead of numbers.  (The Neo4J/Neode folks decided in their infinite wisdom that the default integer type should be a bigint, which doesn't fit in a JavaScript number.  But we really don't need bigints!)

I never imagined the possibility that a database API could be so confusing that putting numbers in doesn't mean getting numbers out, so I burned a ton of time on this yesterday.  I'm probably still cranky from missing sleep—I confess yesterday's experience has caused me to develop a fiery hatred for the deceptive "integer" type!

Reply:
using number